frontend/src/components/features/feedback/likert-scale.tsx

```diff
@@ -207,7 +208,11 @@ export function LikertScale({
               className={cn("text-xl transition-all", getButtonClass(rating))}
               aria-label={`Rate ${rating} stars`}
             >
-              {t(I18nKey.FEEDBACK$STAR_RATING)}
+              {selectedRating && selectedRating >= rating ? (
+                <FaStar />
+              ) : (
+                <FaRegStar />
+              )}
```

Comment:
I think we only need this 🤔 
```suggestion
            <FaStar />
```